README.md

```diff
@@ -1,6 +1,6 @@
 ## Mews - Platform Engineering Team - Interview Test
 
-Please complete and return this technical test before attending your interview at Mews. This isn't something we would like you to spend a long time on - **we recommend no more than 3 hours**. You can give us the results in any way that makes sense to you (if sharing a github private repo, our github usernames are in the footer of the page), but please do not publish them publicly.
+Please complete and return this technical test before attending your interview at Mews. This isn't something we would like you to spend a long time on - **we recommend no more than 3 hours**. You can give us the results in any way that makes sense to you. If sharing a GitHub repository, please make it private and grant access to the GitHub usernames at the bottom of the page.
```

Comment:
in that case we might also need to expand the list of usernames

Reply:
I was thinking we should _reduce_ the number of usernames. It feels painful for the candidate to share their repo with lots of people. It only needs to be shared with a couple of people at Mews so one person can download the zip and share with others. I don't feel strongly about it though.